### PR TITLE
4810 new rnr columns

### LINKED ISSUE #4810: R&R Forms changes
## Is your feature request related to a problem? Please describe 👀

Some requests from the client re the R&R forms.

## Describe the solution you'd like 🎁
I don't want the very specific column name requests to be applied to all users, would it be possible to find a solution which makes the column names customisable in some way? 
Similarly, I'm not certain that all users would require the additional columns. Could these be bespoke?

### Rename columns
* Consumed should be changed to Consumed/Distributed
* Adjusted Consumption should be changed to Adjusted
* AMC (Average Monthly Consumption) should be changed to AMC/AMD (Average Monthly Distribution).

### New columns ( proposed )
* LOSSES Column: Introduce a column for LOSSES, representing quantities removed from the warehouse/system due to issues like expiration, breakage, or damage. This column should be placed before the Adjustment column
* Minimum Quantity Column: Add a Minimum Quantity column positioned just before the Maximum Quantity column. The formula for Minimum Quantity is: Minimum Quantity=AMC/AMD x Minimum Stock Level
* Adjustment Definition: Ensure the definition of adjustments is clearly highlighted to ensure consistent understanding among users.

### System enhancements
- [ ] #5062

### PR DESCRIPTION
**Pull request by lache-melvin**
Fixes #4810

# 👩🏻‍💻 What does this PR do?

Adds new columns to R&R form (losses/min quan)
Screenshot 2024-10-22 at 9 07 05 AM

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] Migrate with existing R&R forms - this is fine
- [ ] Minimum and maximum quantities accurately reflect AMC * min mos/max mos store preferences
- [ ] Losses are included correctly in the final balance

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [x] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1. Update column descriptions
  2.

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/common.json [hunk 1]
```diff
@@ -72,6 +72,10 @@
   "description.program": "Programs are used to manage groups of items, and replenishment schedules.",
   "description.remaining-to-supply": "The quantity remaining to be supplied to the customer which will be re-calculated after a shipment has been created. The calculation is 'Supply - Issued'.",
   "description.requested-quantity": "Quantity requested by customer",
+  "description.rnr-adjustments": "Adjustments made for this item during this period (via Stocktakes or Inventory Adjustments)",
+  "description.rnr-losses": "Manually record losses of this item during the period",
+  "description.rnr-maximum-quantity": "The target quantity of stock to have on hand. =AMC x Store Preference 'Threshold for Overstock'",
+  "description.rnr-minimum-quantity": "The minimum quantity of stock to have on hand. =AMC x Store Preference 'Threshold for Understock'",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/common.json:78]: Added the new/changed ones in common, so hopefully easier to merge into 2.4 develop 😁

**Comment by lache-melvin** [comment]
Docs are done here: https://github.com/msupply-foundation/msupply_docs/pull/112

**Review by jmbrunskill — APPROVED** [review]
Looks great!

File: server/service/src/rnr_form/tests/generate_rnr_form_lines.rs [hunk 2]
```diff
@@ -99,7 +109,9 @@ mod generate_rnr_form_lines {
                 entered_quantity_received: None,
                 entered_quantity_consumed: None,
                 entered_adjustments: None,
-                maximum_quantity: 7.826086956521738, // 2*AMC
+                entered_losses: None,
+                minimum_quantity: 1.9565217391304346, // AMC * months understock (0.5)
```
[the remark this answers is not in the document]
Comment (lache-melvin) [reply server/service/src/rnr_form/tests/generate_rnr_form_lines.rs:113]: Yeah probably... it's displayed as an integer in the frontend, but with the "true" decimal value still available on tooltip. Probably way more accuracy than we really need though 👀 

Think I'll just merge now, but yeah, maybe we can clean up if there is another next time for R&R!

**Pull request merged**